types/lodash/index.d.ts

```diff
@@ -16807,6 +16914,21 @@ declare namespace _ {
             end?: number,
             step?: number
         ): number[];
+
+        /**
+         * This method is like _.range except that it populates values in
+         * descending order.
+         *
+         * @param start The start of the range.
+         * @param index Not used in this overload.
+         * @param guard Enables use as an iteratee for methods like _.map. You should not pass this parameter directly in your code.
+         * @return Returns a new range array.
+         */
+        rangeRight(
+            end: number,
+            index: string | number,
```

Comment:
I'm not understanding the `index - not used in this overload` comments. If it's not used, why have it? Could you provide an example of how it's used?

Reply:
If `guard` is present, lodash ignores the second parameter. Lodash does this so `rangeRight` can be used as an iteratee for `map` (see my other comment).

Reply:
Gotcha. This library always throws me for a loop.